### PR TITLE
Fix missing task info from failed record.log

### PR DESCRIPTION
**Pull request by joepvd**

### CODE REVIEW

**Comment by joepvd** [comment]
From: `Ran 234 tests in 8.211s`
To: `Ran 234 tests in 1.299s`

**Comment by joepvd** [comment]
Moved test speed changes to https://github.com/openshift/doozer/pull/652

File: doozerlib/distgit.py [hunk 1]
```diff
@@ -1035,14 +1035,18 @@ def wait(n):
...
-                    task_id, task_url, nvr = osbs2.build(self.metadata, profile, retries=retries)
-                    record["task_id"] = task_id
-                    record["task_url"] = task_url
-                    record["nvrs"] = nvr
+                    try:
+                        osbs2.build(self.metadata, profile, retries=retries)
+                    except exectools.RetryException:
+                        raise
+                    finally:
+                        record["task_id"] = osbs2.task_id
```
Comment (Ximinhan) [inline doozerlib/distgit.py:1043]: OSBS2Builder object don't have task_id property https://github.com/openshift/doozer/blob/master/doozerlib/osbs2_builder.py#L29

**Pull request merged**